src/printer_lyb.c

```diff
@@ -747,8 +748,8 @@ lyb_print_metadata(struct ly_out *out, const struct lyd_node *node, struct lyd_l
 
     /* with-defaults */
     if (node->schema->nodetype & LYD_NODE_TERM) {
-        if (((node->flags & LYD_DEFAULT) && (lybctx->options & (LYDP_WD_ALL_TAG | LYDP_WD_IMPL_TAG))) ||
-                ((lybctx->options & LYDP_WD_ALL_TAG) && ly_is_default(node))) {
+        if (((node->flags & LYD_DEFAULT) && (lybctx->print_options & (LYDP_WD_ALL_TAG | LYDP_WD_IMPL_TAG))) ||
```

Comment:
Seeing `LYDP_WD_ALL_TAG` here, it should probably be renamed to `LYD_PRINT_WD_ALL_TAG` together with other print options.

Reply:
ok